cmd/validate/image.go

```diff
@@ -127,6 +131,12 @@ func validateImageCmd(validate imageValidationFunc) *cobra.Command {
 			Use an inline EnterpriseContractPolicy spec
 			  ec validate image --image registry/name:tag --policy '{"publicKey": "<path/to/public/key>"}'
 
+			Use a git url for the policy configuration. In this example there should be a file called
+			'policy.yaml' inside a directory called 'default' in the 'github.com/acme-company/ec-configs'
+			git repo. For git repos that are not on GitHub, prefix the url with 'git::'
```

Comment:
Or [gitlab.com](https://github.com/hashicorp/go-getter/blob/main/detect_gitlab.go)

Reply:
Done. (In incoming revision).